### PR TITLE
Improvements and #86 mark

### PR DESCRIPTION
**Pull request by CoMPaTech**
- [x] Moved travis to both 3.7 and 3.8 testing
- [x] Removed asyncio and datetime closing #86 
- [x] `Black`ened and linter adjustments
- [ ] We really should make `connect()` less complex (but not in this PR)
- [x] Recreated test-data
- [x] `dummy`ed left-over unuseds from previous PRs

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def readme():
     license="MIT",
     packages=["Plugwise_Smile"],
     install_requires=[
-        "aiohttp",
+        "asyncio",
```
Comment (bouwew) [inline setup.py]: #86 says:

Found conflicts:
- asyncio
- datetime

Why remove aiohttp and add asyncio?

Comment (CoMPaTech) [reply setup.py]: Yeah, mixup 👎

File: Plugwise_Smile/Smile.py [hunk 2]
```diff
@@ -299,7 +335,6 @@ async def request(
         except etree.XMLSyntaxError:
             _LOGGER.error("Smile returns invalid XML for %s", self._endpoint)
             raise self.InvalidXMLError
-            return None
```
Comment (bouwew) [inline Plugwise_Smile/Smile.py:302]: I need this None to be returned.
Can we change the code so that it's returned properly?

Comment (CoMPaTech) [reply Plugwise_Smile/Smile.py:302]: As discussed on discord: let's review tomorrow (either keep the raise or the return but not both). We're probably after what https://github.com/plugwise/Plugwise-Smile/pull/95/commits/561bea42d10fa9f4e56e9933b4607446815070c4 does, right?

Comment (bouwew) [reply Plugwise_Smile/Smile.py:302]: Yes, correct.

**Review by bouwew — APPROVED** [review]

**Pull request merged**